src/main/java/appeng/recipes/handlers/IWebsiteSerializer.java

```diff
@@ -31,5 +31,5 @@
 
 	String getPattern( RecipeHandler han );
 
-	boolean canCraft( ItemStack output ) throws RegistrationError, MissingIngredientError;
+	boolean canCraft( ItemStack output ) throws RegistrationException, MissingIngredientException;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: appeng.api.exceptions.RegistrationException, appeng.api.exceptions.MissingIngredientException [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1160)